static/src/javascripts/lib/identity.js

```diff
@@ -0,0 +1,5 @@
+// @flow
+
+const identity = (x: any): any => x;
+
+export { identity };
```

Comment:
I remove `lazyload` altogether, it was useless. The identity function is a common idiom in FP, I think it should be part of any codebase.